features/maintenance-mode.feature

```diff
@@ -55,3 +55,30 @@ Feature: Manage maintenance mode of WordPress install.
       """
       Error: Maintenance mode already deactivated.
       """
+
+    When I run `wp maintenance-mode activate`
+    Then STDOUT should be:
+      """
+      Enabling Maintenance mode...
+      Success: Activated Maintenance mode.
+      """
+
+    When I run `wp eval "file_put_contents('.maintenance', '<?php \$upgrading=(time()-601);'); "`
+    And I try `wp maintenance-mode is-active`
+    Then the return code should be 1
+    Then STDERR should contain:
+      """
+      Unable to read the maintenance file timestamp, non-numeric value detected.
+      """
+
+    When I run `wp eval "file_put_contents('.maintenance', '<?php \$upgrading=' . time() + 100 . ';'); "`
```

Comment:
Can we put these three steps in a separate scenario?

Reply:
Absolutely.